api/src/main/java/org/pmiops/workbench/utils/mappers/WorkspaceMapper.java

```diff
@@ -78,42 +77,14 @@ default void setFeaturedWorkspaceInfo(
         isPublished ? featuredWorkspaceService.getFeaturedCategory(dbWorkspace) : null);
   }
 
-  @Mapping(target = "cdrVersionId", source = "cdrVersion")
-  @Mapping(target = "creator", source = "creator.username")
-  @Mapping(target = "etag", source = "version", qualifiedByName = "versionToEtag")
-  @Mapping(
-      target = "googleBucketName",
-      ignore = true) // available via toApiWorkspace(DbWorkspace dbWorkspace, RawlsWorkspaceDetails
-  // fcWorkspace)
-  @Mapping(target = "id", source = "firecloudName")
-  @Mapping(target = "namespace", source = "workspaceNamespace")
-  @Mapping(target = "researchPurpose", source = "dbWorkspace")
-  @Mapping(target = "accessTierShortName", source = "dbWorkspace.cdrVersion.accessTier.shortName")
-  @Mapping(target = "featuredCategory", ignore = true)
-  Workspace toApiWorkspace(DbWorkspace dbWorkspace);
-
   @Mapping(
       target = "accessLevel",
       source = "accessLevel",
       qualifiedByName = "fcToApiWorkspaceAccessLevel")
   WorkspaceResponse toApiWorkspaceResponse(
       Workspace workspace, RawlsWorkspaceAccessLevel accessLevel);
 
-  default WorkspaceResponse toApiWorkspaceResponse(
```

Comment:
removed these 2 to simplify this file.  These are simply compositions of other calls, so I have inlined these as appropriate.